sdk/src/client/stronghold/mod.rs

```diff
@@ -100,7 +100,7 @@ pub struct StrongholdAdapter {
     timeout_task: Arc<Mutex<Option<TaskHandle>>>,
 
     /// The path to a Stronghold snapshot file.
-    pub(crate) snapshot_path: PathBuf,
+    pub snapshot_path: PathBuf,
```

Comment:
Why make this public?

Reply:
I need it in the CLI.

Reply:
I think a getter would be better, because otherwise one can change the path and then end up with multiple stronghold files that only have a part of the data?

Reply:
Agree. Done.